config/datasets/ot_gcp.yaml

```diff
@@ -1,5 +1,5 @@
 # Release specific configuration:
-release_version: "24.03"
+release_version: "24.05"
```

Comment:
24.06?

Reply:
Dummy me, I did not change the calendar 👍🏽